Package.swift

```diff
@@ -24,11 +24,12 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-nio.git", from: "2.10.1"),
         .package(url: "https://github.com/apple/swift-nio-ssl.git", from: "2.0.0"),
         .package(url: "https://github.com/apple/swift-nio-extras.git", from: "1.3.0"),
+        .package(url: "https://github.com/apple/swift-nio-http2.git", from: "1.6.0"),
```

Comment:
do we need http/2 dependencies for this PR?

Reply:
I don’t believe we do!

Reply:
Indeed, we can remove this since this PR only focuses on HTTP/1.1